types/es-value-fixtures/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for es-value-fixtures 1.4
+// Project: https://github.com/ljharb/es-value-fixtures#readme
+// Definitions by: Jordan Harband <https://github.com/ljharb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+
+type ESAbstractDescriptor = {
+    '[[Configurable]]'?: boolean;
+    '[[Enumerable]]'?: boolean;
+} & (
+    | Partial<{
+        '[[Writable]]': boolean;
+        '[[Value]]': unknown;
+    }>
+    | Partial<{
+        '[[Get]]'?: () => unknown;
+        '[[Set]]'?: (value: unknown) => void;
+    }>
+);
+
+type Func = (...args: unknown[]) => unknown;
+
+interface CoercibleObject { valueOf: Func, toString: Func }
+
+declare const fixtures: {
+    primitives: Array<string | number | bigint | boolean | symbol | null | undefined>;
```

Comment:
Are you expected to be able to push/pop from these arrays at runtime? If not, it might be good to annotate them as `ReadonlyArray<T>` or `readonly T[]` to prevent inadvertent mutation.

Reply:
no, i'm not; that's a good idea